src/main/java/uk/gov/indexer/Configuration.java

```diff
@@ -85,10 +85,6 @@ public String getProperty(String key) {
         return Optional.ofNullable(cloudSearchWatermarkEndPoints.get(register));
     }
 
-    public Optional<String> getCTServerEndpointForRegister(String register) {
-        return Optional.ofNullable(properties.getProperty(register + ".ctserver"));
-    }
-
     public Optional<String> elasticSerachEndPoint(String register) {
```

Comment:
just spotted a typo here `Serach`. unrelated to this PR but if you were to fix it I wouldn't mind :)